OpenOversight/app/templates/submit_image.html

```diff
@@ -14,72 +14,75 @@
-            label="dept-select">
-        {{ wtf.render_field(form.department) }}
+      <div class="card mb-5">
+        <div class="card-header">
+          <h2 class="card-title">What department does the officer belong to?</h2>
+        </div>
+        <div class="card-body row">
+          <div class="form-group">
+            <form class="form"
+                  name="dept-select"
+                  id="dept-select"
+                  method="post"
+                  role="form"
+                  label="dept-select">
+              {{ wtf.render_field(form.department) }}
+            </form>
+          </div>
+        </div>
+      </div>
+      <form id="my-cop-dropzone"
+            action="{{ url_for('main.upload', department_id=1) }}"
+            class="dropzone">
       </form>
-      <br>
-    </div>
-    <h3>Drop images here to submit photos of officers:</h3>
-    <form id="my-cop-dropzone"
-          action="{{ url_for('main.upload', department_id=1) }}"
-          class="dropzone">
-    </form>
-    <p>
-      Drag photographs from your computer directly into the box above or click the box to launch a finder window.
-      If you are on mobile, you can click the box above to select pictures from your photo library or camera roll.
-      An image is successfully uploaded when you see a check mark over it.
-    </p>
-    <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
-    <script src="{{ url_for('static', filename='js/dropzone.js') }}"></script>
-    <script src="{{ url_for('static', filename='js/init-dropzone.js') }}"></script>
-    <script>
-        // Select user's preferred department by default
-        document.getElementById("department").selectedIndex = {{ preferred_dept_id }} - 1;
-        // Save the preferred department in dept_id
-        let dept_id = document.getElementById("department").selectedIndex + 1;
-        const csrf_token = "{{ csrf_token() }}";
+      <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
+      <script src="{{ url_for('static', filename='js/dropzone.js') }}"></script>
+      <script src="{{ url_for('static', filename='js/init-dropzone.js') }}"></script>
+      <script>
+          // Select user's preferred department by default
+          document.getElementById("department").selectedIndex = {{ preferred_dept_id }} - 1;
+          // Save the preferred department in dept_id
+          let dept_id = document.getElementById("department").selectedIndex + 1;
+          const csrf_token = "{{ csrf_token() }}";
 
-        // Store changes in drop down list in dept_id variable
-        $(function() {
-            const select_dept = $('#dept-select');
-            select_dept.on('change', function() {
-                // fires when department selection changes
-                dept_id = document.getElementById("department").value;
-            });
-        });
+          // Store changes in drop down list in dept_id variable
```

Comment:
It'd be for later on, but I'd love to get rid of this at some point.

Reply:
Sorry, why would we remove this?

Reply:
We don't need to. The sprinkling of small bits of javascript in the code just feel off to me, that's all.

Reply:
That makes sense! I think we hadn't removed it since the script uses some values injected through Jinja. We could potentially look into refactor that into a static file in a future PR